webknossos-datastore/app/com/scalableminds/webknossos/datastore/datavault/Encoding.scala

```diff
@@ -0,0 +1,18 @@
+package com.scalableminds.webknossos.datastore.datavault
+
+import com.scalableminds.util.enumeration.ExtendedEnumeration
+
+object Encoding extends ExtendedEnumeration {
+  type Encoding = Value
+  val gzip, brotli, none, unsupported = Value
```

Comment:
I’d call it identity rather than none, that matches the http header as well, I think. "" could also default to identity.